docs/getting-started/tutorial-discovering.asciidoc

```diff
@@ -1,18 +1,18 @@
 [[tutorial-discovering]]
 === Discover your data
 
-Using the Discover application, you can enter
+Using the Discover application, enter
 an {ref}/query-dsl-query-string-query.html#query-string-syntax[Elasticsearch
 query] to search your data and filter the results.
 
 . Open *Discover*.
 +
-The current index pattern appears below the filter bar, in this case `shakes*`.
-You might need to click *New* in the menu bar to refresh the data.
+The current index pattern appears below the filter bar, in this tutorial `shakes*` appears.
+To refresh the data, click *New*.
```

Comment:
Not sure this is needed, if in the next step, you are changing the index pattern any way.